protocol/messenger_handler.go

```diff
@@ -265,8 +265,33 @@ func (m *Messenger) PendingNotificationContactRequest(contactID string) (*Activi
 	return m.persistence.ActiveContactRequestNotification(contactID)
 }
 
+func (m *Messenger) createContactRequestForContactUpdate(contact *Contact, messageState *ReceivedMessageState) (*common.Message, error) {
+	contactRequest, err := m.generateContactRequest(
+		messageState.CurrentMessageState.Message.Clock,
+		messageState.CurrentMessageState.WhisperTimestamp,
+		contact,
+		"Please add me to your contacts",
+		false,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	contactRequest.ID = "0x" + types.Bytes2Hex(append(types.Hex2Bytes(contact.ID), 0x20))
```

Comment:
Is this `defaultContactRequestID`?

Reply:
Previously there were two meanings of default contact request, this one left for one created by handling ContactUpdate. Should i return `defaultContactRequestID ` function?

Reply:
fixed